digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/dao/RoadLinkOverrideDAO.scala

```diff
@@ -19,7 +19,7 @@ sealed trait RoadLinkOverrideDAO{
 
   def getValue(linkProperty: LinkProperties) : Int
 
-  def getVVHValue(roadLinkFetched: RoadLinkFetched): Option[Int]
+  def getMasterDataValue(roadLinkFetched: RoadLinkFetched): Option[Int]
```

Comment:
vähän aloin miettimään onkos digiroad tielinkkien master joilloin tää ei voi olla MasterData vaan jokin muu, hyym pitää tarkistaa

Reply:
KGV se ei voi myöskään olla koska siinä on vanhaa dataa, joka pitäisi sitten päivittää vastaamaan KGV jollakin batch jos näin halutaan.

Reply:
hyym tää taitaa sitten hakea dataa linkistä eikä tietokannasta

Reply:
joten tää pitäisi olla getValueFromRoadLinkFetched,vähän pitkä litania

Reply:
mut voidaan mennä tolla ton daily perusteella